javascript-loops-practice/get-student-names.js

```diff
@@ -0,0 +1,19 @@
+/* exported getStudentNames */
+// function getStudentNames(students) {
+//  let key = 2;
+//  const studentNames = [];
+//  for (key in students) {
+//    studentNames.push(Object.values(students[key]));
+//  } console.log(studentNames);
+//  return studentNames;
+// }
+
+function getStudentNames(students) {
+  const studentNames = [];
+  const studentNames2 = [];
```

Comment:
What's the point of this array? We aren't returning it or referencing it anywhere, you're just pushing values here for no reason. I would remove this